backend/btrixcloud/crawlconfigs.py

```diff
@@ -377,7 +366,13 @@ async def update_crawl_config(
         query["modifiedByName"] = user.name
         query["modified"] = dt_now()
 
-        query["profileid"], _ = await self._lookup_profile(update.profileid, org)
+        # if empty str, just clear the profile
+        if isinstance(update.profileid, EmptyStr) or update.profileid == "":
```

Comment:
Seems a little more verbose than needed maybe? Definitely makes intent clear but `if not update.profileid:` should have the same result, no?

Reply:
No, the reason there's a distinction here (and previous slightly confusing logic) is that we have `""` to clear the profile, or `None` to not update it, since the endpoint supports partial updates, and `None` entries are ignored ignored.

Reply:
Ah right!